test-integration/scripts/99-cleanup.sh

```diff
@@ -3,6 +3,8 @@
 set -ex
 
 mv backup.eslintrc.json .eslintrc.json
+mv backup.yo-rc.json .yo-rc.json
```

Comment:
@atomfrede I think you should remove .yo-rc.json from the repository instead.

Reply:
:+1: as long as there aren't negative side effects that would come with doing so.

Reply:
I think the file comes from the blueprint generator to bootstrap the blueprint and can be removed (was not sure about it)